_inc/client/at-a-glance/akismet.jsx

```diff
@@ -77,16 +113,24 @@ const DashAkismet = React.createClass( {
 			return(
 				<DashItem label="Anti-spam (Akismet)" className="jp-dash-item__is-inactive" status="is-warning">
 					<p className="jp-dash-item__description">
-						Whoops! It appears your Akismet key is missing or invalid. <br/>
-						<a href={ akismetSettingsUrl }>Go to Akismet settings to fix</a></p>
+						{ __( 'Whoops! It appears your Akismet key is missing or invalid.' ) }<br/>
+						<a href={ akismetSettingsUrl }>
+							{ __( 'Go to Akismet settings to fix' ) }
+						</a></p>
```

Comment:
I think it would be preferred to put this in a large translation string. Example:

```javascript
__( 'Whoops! It appears your Akismet key is missing or invalid. {{akismetSettings}}Go to Akismet settings to fix{{/akismetSettings}}.', {
		components: {
			akismetSettings:<a href={ akismetSettingsUrl } />
		}
	} )
}
```